packages/amplify-codegen-appsync-model-plugin/src/visitors/appsync-json-metadata-visitor.ts

```diff
@@ -58,7 +62,7 @@ export interface RawAppSyncModelMetadataConfig extends RawAppSyncModelConfig {
    *  plugins:
    *    - amplify-codegen-appsync-model-plugin
    * ```
-   * metaDataTarget: 'javascript'| 'typescript'
+   * metaDataTarget: 'javascript'| 'typescript' | 'typedeclration'
```

Comment:
metaData -> metadata 1 word
typedeclration -> typo, perhaps if camelcase is the convention typeDeclaration